### ISSUE TITLE
bug: wrong colors

### ISSUE THREAD

**Issue by 7flash** [excalidraw/excalidraw#5312]
Take screenshot from daisyui documentation (Shift+Windows+S with default windows Snipping Tool).. Paste into Excalidraw App (Chrome Extension on Desktop Ctrl+V).. Expected to see same colors.. but looks different..

https://daisyui.com/components/stat/#with-custom-colors-and-button

![nineten](https://user-images.githubusercontent.com/4569866/173998932-e785289b-fd4a-4ec3-9f9a-aa6dfd738a60.gif)

**Comment by ad1992** [comment]
Yeah this is happening since you are exporting in dark theme in Excalidraw and we invert and hue-rotate the colors and go with whatever it gives us, this works most of the time. Though I I agree it is confusing specially for the above case and also when you want to use your own custom colors. We need to improve this so colour shades are closer to the actual colors.

**Comment by 7flash** [comment]
Thanks for explaining.. that's right.. switching back to light theme renders correct colors.. what's surprising the issue only happens to that specific color.. didn't notice with any others..

**Comment by anatoliibakhtin** [comment]
it would be cool when inverting colors to also add it for white otherwise you have to play "hide and seek" with them 
<img src="https://user-images.githubusercontent.com/71871141/219973748-9f0e009c-68e7-45e0-a1f8-6431b9ad597e.png" width="40%">

**Comment by dwelle** [comment]
@anatoliibakhtin do you mean when the text is overlaid over an image, it should not be inverted? Not super easy to do for a general case.

**Comment by anatoliibakhtin** [comment]
@dwelle Sorry, I wasn't clear. Added to the comment above.

  As far as I understand, the main function of night mode is to reduce eye strain, when working, at night, secondary is to be beautiful. For me, it is enough to simply invert the white color of the image so that the black text does not disappear into the white background of the image.
So that the colors of the ink and the colors of the image are consistent (do not disappear as they do now) with each other.

**Comment by anatoliibakhtin** [comment]
Sorry, I didn't make my comment above clear. Now Fixed*

**Comment by thiswallz** [comment]
pls solve these issues, it is quite annoying seen the my colors changed on dark mode

**Comment by anatoliibakhtin** [comment]
I love how dark mode - dark color implemented here: https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh
![image](https://github.com/excalidraw/excalidraw/assets/71871141/c67299da-6e2c-4b9f-a9b9-cff2fa94e5be)
Yellow stays yellow and doesn't turn into something like chocolate :))

**Comment by dwelle** [comment]
fixed in https://github.com/excalidraw/excalidraw/pull/10578
